sgf-viewer/src/App.svelte

```diff
@@ -92,7 +92,6 @@
         display: flex;
         justify-content: center;
         padding: 0 min(1.5em, 4vw);
-        width: 100vw;
```

Comment:
not an issue that's new to this PR but i was noticing a horizontal scrollbar on our website, removing this fixes it (not sure why I added it in the first place)